src/ride/track.h

```diff
@@ -448,6 +449,24 @@ enum {
 };
 
 enum {
+	FLAT_TRACK_ELEM_1_X_4_A = 95,
+	FLAT_TRACK_ELEM_2_X_2 = 110,
+	FLAT_TRACK_ELEM_4_X_4 = 111,
+	FLAT_TRACK_ELEM_1_X_5 = 116,
+	FLAT_TRACK_ELEM_1_X_1_A = 118,
+	FLAT_TRACK_ELEM_1_X_4_B = 119,
+	FLAT_TRACK_ELEM_1_X_1_B = 121,
+	FLAT_TRACK_ELEM_1_X_4_C = 122,
+	FLAT_TRACK_ELEM_3_X_3 = 123,
+};
+
+enum {
+	TRACK_FLAG_4 = (1 << 4), // 0x10
+	TRACK_FLAG_5 = (1 << 5), // 0x20
```

Comment:
Seems to indicate that the element connects directly to paths (only used by `1x1` tiles.